spec/controllers/people_controller_spec.rb

```diff
@@ -0,0 +1,111 @@
+# spec/controllers/people_controller_spec.rb
+require 'rails_helper'
+
+RSpec.describe PeopleController, type: :controller do
+  describe "GET #index" do
+    subject { get :index }
+
+    it "responds with 200" do
+      expect(subject.status).to eq(200)
+    end
+    it "renders the template" do
+      expect(subject).to render_template(:index)
+    end
+  end
+
+  describe "GET #show" do
+    subject { get :show, params: { id: person } }
+
+    let(:person) { FactoryGirl.create :person }
+
+    it "renders the template" do
+      expect(subject).to render_template(:show)
+    end
+  end
+
+  describe "GET #new" do
+    subject { get :new }
+
+    it "renders the template" do
+      expect(subject).to render_template :new
+    end
+  end
+
+  describe "GET #edit" do
+    subject { get :edit, params: { id: person } }
+    let(:person) { FactoryGirl.create :person }
+
+    it "renders the template" do
+      expect(subject).to render_template :edit
+    end
+  end
+
+  describe "POST #create" do
+    subject { post :create, params: { person: params } }
+    let(:params) { FactoryGirl.attributes_for(:person) }
+
+    it "changes person count by +1" do
+      expect { subject }.to change(Person, :count).by(1)
+    end
+
+    it "loads the person into @person" do
```

Comment:
Not sure this is the best description for the spec. Seems like an artifact from when you were testing `assigns`